packages/preload-webview/src/webview-preload.spec.ts

```diff
@@ -20,6 +20,7 @@
 
 import { afterEach } from 'node:test';
 
+import type { WebviewApi } from '@podman-desktop/webview-api';
```

Comment:
Webview is UI, why not to have it as a part of packages/ui and have it in code like

```
import type { WebviewApi } from '@podman-desktop/ui-svelte';
```
?

Reply:
you don't need the UI package to work with Webviews